server/api/game/controller.js

```diff
@@ -0,0 +1,124 @@
+const uuidv1 = require('uuid/v1');
```

Comment:
Have you guys not setup the babel? If no please set it up, you can then use import/export es6 features

Reply:
Added babelrc and updated the server code.